src/comm/QGCMAVLink.cc

```diff
@@ -113,6 +113,8 @@ QGCMAVLink::VehicleClass_t QGCMAVLink::vehicleClass(MAV_TYPE mavType)
     case MAV_TYPE_QUADROTOR:
     case MAV_TYPE_COAXIAL:
     case MAV_TYPE_HELICOPTER:
+    case MAV_TYPE_SUBMARINE:
```

Comment:
This is right in the middle of the multirotor block. It's going to return submarine for the multi-rotor type above it.

Reply:
Oops!
I moved it under boat now.